### PR TITLE
Implemented get/set_default_isolation_level() functions

### PR DESCRIPTION
**Pull request by volka**
... to set the transaction isolation level default per-connection.

### CODE REVIEW

File: tests/Select.cpp [hunk 1]
```diff
@@ -183,8 +183,22 @@ int Select(int, char* [])
...
   }
 
-  auto transaction = start_transaction(db, sqlpp::isolation_level::read_committed);
-  std::cout << (db._mock_data._last_isolation_level == sqlpp::isolation_level::read_committed) << std::endl;
-  
+  {
+      auto transaction = start_transaction(db, sqlpp::isolation_level::read_committed);
+      if (db._mock_data._last_isolation_level != sqlpp::isolation_level::read_committed)
+      {
+          std::cout << "Error: transaction isolation level does not match expacted level" << std::endl;
```
Comment (rbock) [inline tests/Select.cpp]: s/expacted/expected/ :-)

**Comment by rbock** [comment]
I like this much better :-)

File: tests/Select.cpp (hunk 1 above)
Comment (volka) [inline tests/Select.cpp]: Nice :) Corrected

**Comment by rbock** [comment]
Merged and released (0.49)

**Pull request merged**